scripts/compile.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+source "$(dirname "${BASH_SOURCE[0]}")/lib.sh"
+
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo add grafana https://grafana.github.io/helm-charts
+helm --repository-cache /tmp/cache repo update
+# Check for unexpected changes.
+# Helm dependencies are versioned using ^ which accepts minor & patch changes:
+# 	e.g. ^1.2.3 is equivalent to >= 1.2.3 < 2.0.0
+helm dependency update coder-observability/
+# We *expect* that the versions will change in the rendered template output, so we ignore those, but
+# if there are changes to the manifests themselves then we need to fail the build to force manual review.
+helm template -f coder-observability/values.yaml coder-observability coder-observability/ | \
```

Comment:
Specify `--namespace` otherwise your existing `kubeconfig` will influence the output here!

Reply:
Good call! I thought `helm template` was setting it from the release name, but alas not.